cmd/memberagent/main.go

```diff
@@ -38,7 +38,7 @@ import (
 	fleetv1alpha1 "go.goms.io/fleet/apis/v1alpha1"
 	imcv1alpha1 "go.goms.io/fleet/pkg/controllers/internalmembercluster/v1alpha1"
 	imcv1beta1 "go.goms.io/fleet/pkg/controllers/internalmembercluster/v1beta1"
-	workapi "go.goms.io/fleet/pkg/controllers/work"
+	workapi "go.goms.io/fleet/pkg/controllers/workv1alpha1"
```

Comment:
If we have the new work controller in the work directory and move the old to workv1alpha1 then none of those changes are needed, right?

It seems that the next PR will revert all those changes